Sources/Orbit/Foundation/Spacing/Spacing.swift

```diff
@@ -7,46 +7,61 @@ import CoreGraphics
 /// - Note: [Orbit definition](https://orbit.kiwi/foundation/spacing/)
 public enum Spacing {
     /// 2 pts.
-    public static let xxxSmall: CGFloat = 2
+    case xxxSmall
     /// 4 pts.
-    public static let xxSmall: CGFloat = 4
+    case xxSmall
     /// 8 pts.
-    public static let xSmall: CGFloat = 8
+    case xSmall
     /// 12 pts.
-    public static let small: CGFloat = 12
+    case small
     /// 16 pts.
-    public static let medium: CGFloat = 16
+    case medium
     /// 20 pts.
-    public static let xMedium: CGFloat = 20
+    case xMedium
     /// 24 pts.
-    public static let large: CGFloat = 24
+    case large
     /// 32 pts.
-    public static let xLarge: CGFloat = 32
+    case xLarge
     /// 44 pts.
-    public static let xxLarge: CGFloat = 44
+    case xxLarge
     /// 60 pts.
-    public static let xxxLarge: CGFloat = 60
+    case xxxLarge
+
+    public var value: CGFloat {
```

Comment:
If this is `public`, why not just make it the raw value of `Spacing`? What do we get from `.value` vs `.rawValue`?

Reply:
Now I remember - it was originally due to multiple "cases" that had the same value. Updated 👍

Reply:
I think I even saw something like this used for such cases...

```swift
enum Foo: Int {
    case someValue = 2
    case anotherValue = 3

    static var sameAsTwo: Foo { .someValue }
}
```
... but if we don't need that anymore, even better.

Reply:
True, that could be done if we ever need it again. But for now, it can be a simple enum